src/modules/10-common/navigation/MainNav/__tests__/MainNav.test.tsx

```diff
@@ -104,12 +104,11 @@ describe('main nav tests', () => {
     expect(queryByText('featureFlagsText')).not.toBeNull()
   })
 
-  test('when ng dashbpard is enabled', () => {
+  test('when ng dashb0ard is enabled', () => {
```

Comment:
One typo replaced by another typo lol
```suggestion
  test('when ng dashboard is enabled', () => {
```

Reply:
OOPS